templates/style_guides/ruby.yml

```diff
@@ -1,41 +1,41 @@
-AllCops:
-  Include:
-    - 'Gemfile'
-    - 'Rakefile'
-  Exclude:
-    - 'bin/*'
-    - 'db/schema.rb'
-    - 'db/migrate/*'
-
-ParameterLists:
+Metrics/ParameterLists:
+  Description: Avoid parameter lists longer than three or four parameters.
+  Enabled: true
   Max: 4
   CountKeywordArgs: true
 
-ClassLength:
+Metrics/ClassLength:
+  Description: Avoid classes longer than 100 lines of code.
+  Enabled: true
+  CountComments: false
   Max: 100
 
-LineLength:
-  Enabled: 100
+Metrics/LineLength:
+  Description: Limit lines to 100 characters.
+  Enabled: true
+  Max: 100
 
-MethodLength:
+Metrics/MethodLength:
+  Description: Avoid methods longer than 15 lines of code.
+  Enabled: true
   CountComments: false
   Max: 15
 
-HashSyntax:
-  EnforcedStyle: ruby19
-
-EmptyLinesAroundAccessModifier:
+Metrics/BlockNesting:
+  Description: Avoid excessive block nesting
   Enabled: true
+  Max: 3
 
-EndAlignment:
-  AlignWith: variable
-
-CaseIndentation:
-  IndentWhenRelativeTo: end
-  IndentOneStep: false
+Metrics/AbcSize:
+  Description: A calculated magnitude based on number of assignments, branches, and conditions.
+  Enabled: false
 
-StringLiterals:
+Style/StringLiterals:
+  Description: Checks if uses of quotes match the configured preference.
+  Enabled: true
   EnforcedStyle: single_quotes
```

Comment:
About that : 
I'm wondering if we should still prefer single quotes in our guidelines.
I used to prefer that, but today I think it would be better to prefer double, for 2 reasons essentially :
- The first one is coherence. By default, Rails generated stuff are using double quotes. Generating a new rails app, a gem, new migration, `db/schema.rb` are always using double quotes.
- The second one is for inside variables. No need anymore to change quotes styles when we want to add a variable inside, so no more insight question like "do I need to show a variable in this string?" -> This is in adequation with "Don't make me think" book ;)

An article on performance about double vs single quotes : 
http://viget.com/extend/just-use-double-quoted-ruby-strings